code/modules/projectiles/guns/specialist.dm

```diff
@@ -548,6 +548,20 @@ Note that this means that snipers will have a slowdown of 3, due to the scope
 	placed_overlay_iconstate = "pepper"
 
 //-------------------------------------------------------
+
+/obj/item/weapon/gun/rifle/pepperball/pepperball_mini	//TODO: firemode currently has to be changed before attaching the gun
```

Comment:
firemode changing is something i have to add support for. If you are capable thatd be much appreciated.